# Face bounding box with marker positions
Header header
Rectangle boundingbox
Vector2D[] landmark
float32 probability
